vm/isolated/isolated.go

```diff
@@ -324,14 +363,50 @@ func (inst *instance) Run(timeout time.Duration, stop <-chan bool, command strin
 	if inst.debug {
 		tee = os.Stdout
 	}
-	merger := vmimpl.NewOutputMerger(tee)
-	merger.Add("dmesg", dmesg)
-	merger.Add("ssh", rpipe)
+	inst.merger = vmimpl.NewOutputMerger(tee)
+	inst.merger.Add("dmesg", dmesg)
+	inst.merger.Add("ssh", rpipe)
 
-	return vmimpl.Multiplex(cmd, merger, dmesg, timeout, stop, inst.closed, inst.debug)
+	return vmimpl.Multiplex(cmd, inst.merger, dmesg, timeout, stop, inst.closed, inst.debug, false)
 }
 
+func (inst *instance) readPstoreContents() ([]byte, error) {
+	log.Logf(0, "reading pstore contents")
+	dir, err := ioutil.TempDir("", "pstore")
```

Comment:
Since we only need 1 file, it would be more reasonable to create a temp file with osutil.TempFile and scp into this file.
I would also try to execute cat on the pstore file with ssh and then read Stdout directly into memory, this would not require any temp files at all.

Reply:
Thank you for the review! I'll make the above changes and update the PR soon.